src/parse.ts

```diff
@@ -0,0 +1,18 @@
+const regex = /^(?<key>[^:\s\n][^:\n]*)\s*:\s*(?<value>[^\n]+)$/gm
```

Comment:
Do you think that the pattern of having a settings file with the parsing rules would be better? For example: https://github.com/froi/add_invite_user/blob/a36c6d931353c89fdfcdccd7105927cb425de51f/src/main.js#L34

That way changes in the way a user writes an issue won't need a code change.